test/testpost.jl

```diff
@@ -0,0 +1,29 @@
+@testset "Test e4st_post" begin
+    # First make a few runs
+    nsims = 3
+    sim_paths = map(1:nsims) do i
+        run_e4st(
+            joinpath(@__DIR__, "config/config_3bus.yml")
+        )
+    end
+
+    out_path = joinpath(@__DIR__, "out/post")
+    
+    rm(out_path, force=true, recursive=true)
+    @test ~isdir(out_path)
+
+    post_config = read_post_config(
+        joinpath(@__DIR__, "config/post_config.yml");
```

Comment:
so just checking my understanding, to do the post processing you would either write a separate script or add to the bottom of the run script, and call e4st_post on the post config